lib/rb_shift/project.rb

```diff
@@ -60,54 +60,64 @@ def routes(update = false)
     end
 
     def create_secret(name, kind, **opts)
+      log.info "Creating secret #{kind} #{name}"
       execute "create secret #{kind} #{name}", **opts
       secrets true if @_secrets
     end
 
     def add_role_to_user(role, username)
+      log.info "Adding role to user #{role} #{username}"
       execute "policy add-role-to-user #{role} #{username}"
       role_bindings true if @_role_bindings
     end
 
     def add_role_to_group(role, groupname)
+      log.info "Adding role to group #{role} #{groupname}"
       execute "policy add-role-to-group #{role} #{groupname}"
       role_bindings true if @_role_bindings
     end
 
     def create_template(file)
+      log.info "Creating template from file: #{file}"
       execute "create -f \"#{file}\""
       templates true if @_templates
     end
 
     def create_config_map(name, source, path, **opts)
+      log.info "Creating config map: #{name}"
       execute "create configmap #{name}", source.to_sym => path, **opts
       config_maps true if @_config_maps
     end
 
     def create_service(name, kind, **opts)
+      log.info "Creating service: #{kind} #{name}"
       execute "create service #{kind} #{name}", **opts
       services true if @_services
     end
 
     def delete(block = false, timeout = 1)
+      log.info "Deleting project #{@name}"
       execute "delete project #{@name}"
       @client.wait_project_deletion(@name, timeout) if block
     end
 
     def wait_for_deployments(timeout = 30, update = false)
+      log.debug 'Waiting for deployments...'
       deployment_configs true
       wait = true
       while wait
         sleep timeout
         wait = !deployment_configs(update).select(&:running?).empty?
       end
+      log.debug 'Deployment finished'
```

Comment:
This is more information than debug message and also there are many of them :) .